redasql/result_formatter.py

```diff
@@ -81,9 +81,11 @@ def _format_result_to_row_base(self):
 
     def _format_result_to_column_base(self):
         row_for_tables = []
-        for row in self.rows:
+        for i, row in enumerate(self.rows, 1):
             for column_name in self.column_name_list:
                 row_for_tables.append([column_name, self._replace_line_break(row[column_name])])
+            if i < len(self.rows):
+                row_for_tables.append(['-' * 5, '-' * 5])
```

Comment:
plz, `'-' * 5`  move to `constants.py`.
This variable doesn't have to be generated every time

Reply:
That's right!

I fixed it.
https://github.com/denzow/redasql/pull/38/commits/784b1daae1f346733b29b1053539ba506fc220f1